packages/contentful--create-contentful-app/src/logger.ts

```diff
@@ -0,0 +1,28 @@
+import chalk from 'chalk';
+
+export function warn(message: string): void {
+  console.log(`${chalk.yellow('Warning:')} ${message}`);
+}
+
+export function error(message: string, error: string): void {
+  console.log(`${chalk.red('Error:')} ${message}
+
+  ${error.startsWith('Error: ') ? error.substring(7) : error}
+`);
+}
+
+export function highlight(str: string) {
+  return chalk.bold(str);
+}
+
+export function choice(str: string) {
+  return chalk.greenBright(str);
+}
+
+export function success(str: string) {
+  return chalk.cyan(str);
```

Comment:
Maybe make success `greenBright` and 'choice' - cyan?

Reply:
I have actually used the existing color scheme (which I am not sure how was chosen). Maybe @fabe has an idea or opinions?

Reply:
It was my who added greenBright as a fallback choice color, but that was pretty random.

Reply:
Maybe that can be addressed separately anyway. This branch includes a lot of changes already. WDYT?